crates/interpreter/src/exec.rs

```diff
@@ -1407,19 +1382,18 @@ struct Frame<'m> {
     arity: usize,
     ret: &'m [u8],
     locals: Vec<Val>,
-    labels: Vec<Label>,
     side_table: &'m [SideTableEntry],
     /// Total length of the value stack in the thread prior to this frame.
     prev_stack: usize,
+    labels_cnt: usize,
```

Comment:
@ia0 Do you expect this to be removed eventually? Only from the performance perspective, I'm not sure how much gain there would be by removing it. WDYT?

Reply:
Yes, I expect this to be eventually removed (as well as `push_label()`). This is not only cleaner, but also useful for when we'll write a compiler to our own bytecode (or an ISA) because it means we don't need to compile instructions like `Block` or `Loop` that don't do anything except indicate a scope.

But this can be done in a separate PR.